lib/stripe.js

```diff
@@ -173,10 +197,32 @@ Stripe.prototype = {
       // URI-encode in case there are unusual characters in the system's uname.
       userAgent.uname = encodeURIComponent(uname) || 'UNKNOWN';
 
+      if (self._appInfo) {
+        userAgent.application = self._appInfo;
+      }
+
       cb(JSON.stringify(userAgent));
     });
   },
 
+  getAppInfoAsString() {
```

Comment:
It looks like this is not working on older Node versions. I guess something like `getAppInfoAsString: function() {` would work?